octopoes/octopoes/repositories/ooi_repository.py

```diff
@@ -292,9 +301,21 @@ def list_oois(
         limit: int = 20,
         scan_levels: set[ScanLevel] = DEFAULT_SCAN_LEVEL_FILTER,
         scan_profile_types: set[ScanProfileType] = DEFAULT_SCAN_PROFILE_TYPE_FILTER,
+        search_string: str | None = None,
+        order_by: Literal["scan_level", "object_type"] = "object_type",
```

Comment:
Could we also sort on the id? Im not seeing a reason why not.

Reply:
We could! But the id always starts with object type so it does not make a difference. Also, the ID is not shown in the table

Reply:
ah check, true. Sorting would then be done on the human readable, which the database does not 'know'